make superposm.iso errors

followed guidance in readme and was running on a fresh Amazon EC2 t2.xlarge running Ubuntu Server 18.04 with 60GB storage
```
7d5ceabd8a55: Pull complete
1236d99e9c40: Pull complete
27a00fbe6b0a: Pull complete
Digest: sha256:6eb84e4372ced5f3c37339a7cf82b9880d9905210fb96fe6bdcaee6b0bc3b155
Status: Downloaded newer image for opendronemap/nodeodm@sha256:6eb84e4372ced5f3c37339a7cf82b9880d9905210fb96fe6bdcaee6b0bc3b155
docker.io/opendronemap/nodeodm@sha256:6eb84e4372ced5f3c37339a7cf82b9880d9905210fb96fe6bdcaee6b0bc3b155
Created symlink /etc/systemd/system/multi-user.target.wants/nodeodm.service → /etc/systemd/system/nodeodm.service.
Disk usage after nodeodm:
Filesystem      Size  Used Avail Use% Mounted on
/dev/loop2       14G   13G  824M  95% /
none            492K     0  492K   0% /dev
udev            7.9G     0  7.9G   0% /dev/fuse
tmpfs           100K     0  100K   0% /dev/lxd
tmpfs           100K     0  100K   0% /dev/.lxd-mounts
tmpfs           7.9G  8.0K  7.9G   1% /dev/shm
tmpfs           7.9G  1.9M  7.9G   1% /run
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           7.9G     0  7.9G   0% /sys/fs/cgroup
[Thu Aug  1 16:46:50 2019] Bootstrap complete: 0
lxc snapshot $(cat container) nodeodm
Error: subvolume snapshot failed, source=/var/lib/lxd/storage-pools/default/containers/evolving-jay, dest=/var/lib/lxd/storage-pools/default/snapshots/evolving-jay/nodeodm, output=ERROR: cannot snapshot '/var/lib/lxd/storage-pools/default/containers/evolving-jay': No space left on device
Create a snapshot of '/var/lib/lxd/storage-pools/default/containers/evolving-jay' in '/var/lib/lxd/storage-pools/default/snapshots/evolving-jay/nodeodm'

Makefile:118: recipe for target 'nodeodm' failed
make: *** [nodeodm] Error 1
```

--- a/live/README.md
+++ b/live/README.md
@@ -44,16 +44,16 @@ access services that were started.
 
 ## ISO Creation
 
-We've tested using Amazon EC2 t2.xlarge (for 16GB RAM) running Ubuntu Server 18.04 with 60GB disk storage.
+We've tested using Amazon EC2 m5a.2xlarge (for 32GB RAM) running Ubuntu Server 18.04 with 100GB disk storage.
 To setup and install dependencies, run:
+
 ```bash
-git clone https://github.com/posm/posm-build.git
-cd posm-build
-git submodule update --init 
-cd subiquity && make install_deps
-cd ../
-sudo apt install make lxd pigz snapcraft p7zip-full xorriso isolinux
-lxd init  # accepting all defaults
+sudo apt update && sudo apt install lxd make p7zip-full pigz snapcraft xorriso isolinux
+sudo lxd init --auto --storage-backend dir
+
+git clone --recursive https://github.com/posm/posm-build.git
+cd posm-build/live
+cd subiquity && make install_deps && cd ..
 ```
 
 To create a Live Installer ISO, run:

